File: SpielterminApi/Models/SpielerDto.cs
namespace SpielterminApi.Models
{
    public class SpielerDto
    {
        public required string Username { get; set; } =string.Empty;
        public required string Password { get; set; } = "$2a$11$KWZCcnaR3ZeUTl/iEfoGWuITJcO/.rFw/kCWenOhwD3H1JwQHN1Ni";
        public string? Vorname { get; set; } = string.Empty;

        public string? Nachname { get; set; } = string.Empty;

        public string? Wohnort { get; set; } = string.Empty;

        public string? Straße { get; set; } = string.Empty;

        public string? Hausnummer { get; set; } = string.Empty;
    }
}

File: Frontend/BoardGameApp/app/src/main/java/com/example/boardgameapp/DTO/SpielerDto.java
package com.example.boardgameapp.DTO;

public class SpielerDto {
    private int id;
    private String username;
    private String password;
    private String vorname;
    private String nachname;
    private String wohnort;
    private String straße;
    private String hausnummer;
    private String plz;
    private int verspätung;

    public SpielerDto() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getVorname() {
        return vorname;
    }

    public void setVorname(String vorname) {
        this.vorname = vorname;
    }

    public String getNachname() {
        return nachname;
    }

    public void setNachname(String nachname) {
        this.nachname = nachname;
    }

    public String getWohnort() {
        return wohnort;
    }

    public void setWohnort(String wohnort) {
        this.wohnort = wohnort;
    }

    public String getStraße() {
        return straße;
    }

    public void setStraße(String straße) {
        this.straße = straße;
    }

    public String getHausnummer() {
        return hausnummer;
    }

    public void setHausnummer(String hausnummer) {
        this.hausnummer = hausnummer;
    }

    public String getPlz() {
        return plz;
    }

    public void setPlz(String plz) {
        this.plz = plz;
    }

    public int getVerspätung() {
        return verspätung;
    }

    public void setVerspätung(int verspätung) {
        this.verspätung = verspätung;
    }
}
